Kernel/platform/platform-rpipico/misc.c

```diff
@@ -28,9 +28,9 @@ void plt_reboot(void)
 
 void plt_monitor(void)
 {
-    sleep_ms(1); // wait to print any remaining messages
```

Comment:
I removed the sleep here, because plt_monitor could be called from interrupt.
The reason I added it is for kernel to print panic properly, but now tty1 is fully blocking.